### ISSUE TITLE
Azure Load Testing

### ISSUE THREAD

**Issue by jascam** [jascam/CodePlexFoo#522]
I am looking for a sample to get my load testing task off the ground. One tutorial that was fairly helpful is:
http://geekswithblogs.net/TarunArora/archive/2011/11/27/part-2ndashload-testing-in-the-cloud.aspx 
 
But I failed to get the domain join to work, so I couldn't connect the test agent to an on premise controller. As a side note, I am not even sure what the connection between the 2 allows.
 
Another route I was looking into (but I am not sure it is a very good route) is to package up my test dll, and run the tests as a startup task. However, the only thing I am worried about there is that I don't know if the roles will be able to run mstest from the command line.
 
In summary, I have automated UI tests (using Selenium) that exist in Visual Studio and are run using mstest. I want to run them concurrently to effectively load test them using Azure, and have no idea how to do this.

#### Migrated CodePlex Work Item Details
CodePlex Work Item ID: '10664'
Vote count: '3'
